CHANGES.md

```diff
@@ -1,7 +1,12 @@
 
 ## Changelog
 
-### 1.3.1
+### 1.3.2 (Unreleased)
+ * Fixing python 3 compatibility in Simple HTTP Server fixture
+ * Fixed broken tests in pytest-profiling
+ * Removed pytest-fixture-config decorator 'requires_config' and 'yield_requires_config'
```

Comment:
Can you give the reason why here - this is due to imcompatibility with new versions of pytest